src/styles/utils/variables.scss

```
@@ -0,0 +1,14 @@
+$bg-header: #FFF;
+$shadow-header: rgba(0, 0, 0, 0.05);
+$nav-link-color: #060B35;
+$bg-is-active: #00ACDC;
+$title-color: #060B35;
```

Why do you create two variables with the same values?
```suggestion
$nav-link-color: #060B35;
$bg-is-active: #00ACDC;
```